src/test/app/claims/CCJModelConverter.ts

```diff
@@ -18,12 +32,178 @@ const ccjDraft = new DraftCCJ().deserialize({
+    const claim: Claim = new Claim().deserialize(claimStoreMock.sampleClaimIssueObj)
+    const countyCourtJudgment: CountyCourtJudgment = CCJModelConverter.convertForRequest(draft, claim)
     expect(countyCourtJudgment).to.be.deep.equal(new CountyCourtJudgment(undefined, PaymentOption.IMMEDIATELY, undefined, undefined, undefined, undefined, CountyCourtJudgmentType.DEFAULT))
   })
 
+  it('should convert to CCJ - for a valid CCJ draft for full admission response paying by set date on breach of payment terms', () => {
+    const draft: DraftCCJ = ccjDraftWithDefendantDOBKnown
+    const claim: Claim = new Claim().deserialize(sampleClaimWithFullAdmissionWithSetDateResponseObj)
+    const DOB: Moment = dob.toMoment()
+    const countyCourtJudgment: CountyCourtJudgment = CCJModelConverter.convertForRequest(draft, claim)
+    expect(countyCourtJudgment).to.be.deep.equal(new CountyCourtJudgment(DOB, PaymentOption.IMMEDIATELY, undefined, undefined, undefined, undefined, CountyCourtJudgmentType.ADMISSIONS))
+  })
+
+  it('should convert to CCJ - for a valid CCJ draft for full admission response paying by installments on breach of payment terms', () => {
+    const draft: DraftCCJ = ccjDraftWithInstallments
+    const claim: Claim = new Claim().deserialize(sampleClaimWithFullAdmissionWithInstallmentsResponseObj)
+    const expectedRepaymentPlan: RepaymentPlan = new RepaymentPlan(100, new LocalDate(2010, 12, 30).toMoment(), 'EACH_WEEK')
+    const DOB: Moment = dob.toMoment()
+    const countyCourtJudgment: CountyCourtJudgment = CCJModelConverter.convertForRequest(draft, claim)
+    expect(countyCourtJudgment).to.be.deep.equal(new CountyCourtJudgment(
+      DOB,
+      PaymentOption.INSTALMENTS,
+      undefined,
+      expectedRepaymentPlan,
+      undefined,
+      undefined,
+      CountyCourtJudgmentType.ADMISSIONS)
+    )
+  })
+}
+)
+
+describe('CCJModelConverter - Unit test on ModelConverter', () => {
+  const sampleClaimWithInstalments = {
+    ...claimStoreMock.sampleClaimIssueObj,
+    response: fullAdmissionResponseWithInstallmentsAndPaymentDateElapsed,
+    settlement: {
+      partyStatements: [
+        {
+          type: StatementType.OFFER.value,
+          madeBy: MadeBy.DEFENDANT.value,
+          offer: {
+            content: 'My offer contents here.',
+            completionDate: '2020-10-10',
+            ...repaymentPlanPaymentIntention
+          }
+        },
+        {
+          madeBy: MadeBy.CLAIMANT.value,
+          type: StatementType.ACCEPTATION.value
+        }
+      ]
+    },
+    settlementReachedAt: new LocalDate(2010, 1, 1).toMoment()
+  }
+  it('should get undefined CCJPaymentOption when response not present', () => {
+    const claim: Claim = new Claim().deserialize(claimStoreMock.sampleClaimIssueObj)
+    const ccjPaymentOption: CCJPaymentOption = retrievePaymentOptionsFromClaim(claim)
+    expect(ccjPaymentOption).to.be.equal(undefined)
```

Comment:
minor: could use instead `.to.be.undefined`, here and elsewhere

Reply:
Added ignore and changed it as requested.